crates/biome_graphql_parser/src/lexer/mod.rs

```diff
@@ -715,6 +716,15 @@ impl<'src> GraphqlLexer<'src> {
         }
         COMMENT
     }
+
+    fn consume_commas(&mut self) -> GraphqlSyntaxKind {
```

Comment:
I'm wondering if we need to treat a sequence of commas as a single comma token. Is this according to a specification?

Reply:
Arcording to the spec commas is insignificant, similar to whitespace or newline, so I just bundle them up to skip all of them
https://spec.graphql.org/October2021/#sec-Insignificant-Commas

Reply:
I think @denbezrukov means we could bundle commas similar to how whitespace gets bundled. Personally I doubt it matters very much, since I wouldn't expect humans to write GraphQL with a lot of consecutive commas, even if it might happen occassionally.

Reply:
I wonder if, in this case, commas should be part of trivia